### PR TITLE
[IMP] micro_brewery: add deposit and tax

### PR DESCRIPTION
**Pull request by jaeschwa**
This commit aims at improving the micro_brewery module. Including the changes from the beverage distributor module. This is related to the deposit, composition of the bin that can be decomposed. The automatic creation of bills of materials and the reordering rules.

task-4266461

### CODE REVIEW

File: micro_brewery/data/product_product.xml [hunk 1]
```diff
@@ -469,6 +469,23 @@
...
+        <field name="is_storable" eval="True"/>
+        <field name="purchase_method">receive</field>
+        <field name="service_type">manual</field>
+        <field name="invoice_policy">order</field>
+        <field name="product_pricing_ids" eval="[(6,0,[ref('product_pricing_1')])]"/>
+        <field name="rent_ok" eval="True"/>
+        <field name="extra_daily" eval="10.0"/>
+        <field name="self_order_available" eval="False"/>
+        <field name="preparation_time">24.0</field>
+        <field name="website_sequence">10165</field>
```
Comment (vava-odoo) [inline micro_brewery/data/product_product.xml:486]: Shouldn't it be published? (in demo)

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Thanks. Only reviewed the second commit for now. Could you fix it?
I'm not sure yet if I'll do a second round of review afterwards, or update it myself.

File: micro_brewery/data/knowledge_article.xml [hunk 2]
```diff
@@ -1,200 +1,287 @@
...
+                <p>Want to know more about unit of measures ?</p>
+                <p><a class="btn btn-secondary flat" href="https://www.odoo.com/documentation/latest/applications/inventory_and_mrp/inventory/product_management/configure/uom.html?highlight=unit%20measure">🎓 Units of measure</a></p>
+            </div>
+        </div>
+        <p><br /></p>
+        <p>
+            <strong><span class="h3-fs">Rental</span></strong>
+        </p>
+        <p>If you want to rent Beer Taps to your customer, you can explore the rental App and discover how you can manage your rental schedule easily and provide a great extended service to your customers.</p>
+        <p><br /></p>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply micro_brewery/data/knowledge_article.xml:175]: Sorry but I am not clear, that what is the use of adding rental app and rental product if there is no mention in the knowledge article ?

File: micro_brewery/data/knowledge_article.xml [hunk 3]
```diff
@@ -1,200 +1,287 @@
...
-</div>
-<p> <br/></p>
-<h2><strong>1. Crafting a beer</strong></h2>
- <hr/>
-<p><strong><span class="h3-fs">Create a Recipe </span></strong></p>
-<p>In the manufacturing application, you can create a bill of materials to define how the beer should be crafted. </p>
-<ol>
-    <li><strong>Indicate what will be the finished product </strong>at the end of your recipe and the quantity that should be produced with the recipe. for example:</li>
-    <li class="oe-nested">
+        <h1 style="font-size: 54px;" data-oe-version="1.0">Odoo for Microbrewery</h1>
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply micro_brewery/data/knowledge_article.xml]: It was because I have formatted the knowledge article but now I have reverted the change

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have updated the changes 
Thank You :innocent:

File: micro_brewery/data/product_product.xml (hunk 1 above)
Comment (dhrs-odoo) [reply micro_brewery/data/product_product.xml:486]: No it is okay

**Comment by vava-odoo** [comment]
@robodoo squash

**Review by vava-odoo — APPROVED** [review]

**Pull request closed without merging**